packages/react-components/react-swatch-picker-preview/etc/react-swatch-picker-preview.api.md

```diff
@@ -96,19 +93,30 @@ export const swatchPickerCSSVars: {
     gridGap: string;
 };
 
+// @public (undocumented)
+export type SwatchPickerGridProps = {
+    items: SwatchProps[];
+    columnCount: number;
+    renderRow?: (props: {
+        children: JSX.Element[];
+        rowId: string | number;
+    }) => JSX.Element;
+    renderSwatch?: (item: SwatchProps) => JSX.Element;
+};
+
 // @public (undocumented)
 export type SwatchPickerOnSelectEventHandler = EventHandler<SwatchPickerOnSelectionChangeData>;
 
 // @public (undocumented)
 export type SwatchPickerOnSelectionChangeData = EventData<'click', React_2.MouseEvent<HTMLButtonElement>> & {
     selectedValue: string;
-    selectedColor: string;
+    selectedSwatch: string;
```

Comment:
I'm thinking whether we need to add separately `selectedColor` and `selectedImage` for cases when colors and used together with images